### ISSUE TITLE
Is this board compatible with Arduino framework?

### ISSUE THREAD

**Issue by AlexC-0806** [vcc-gnd/YD-ESP32-S3#18]
Hello,

I recently purchased a board from Aliexpress and have been searching for relevant documentation. I noticed that the provided firmware on this GitHub page is exclusively for MicroPython.

My primary concern is whether this board is compatible with the Arduino framework and, more specifically, if it can be programmed via the PlatformIO IDE in Visual Studio Code. Are there any special configurations required to facilitate this?

Additionally, I believe this board should technically be identical to the one detailed here: https://docs.espressif.com/projects/esp-idf/en/latest/esp32s3/hw-reference/esp32s3/user-guide-devkitc-1.html. Given this, I assume there should be no issues programming it using Visual Studio Code and PlatformIO.

Could anyone confirm these assumptions or provide further guidance? I am particularly interested in understanding the exact compatibility of this board with the PlatformIO and Arduino framework.

Thank you in advance for your assistance.

**Comment by robertlipe** [comment]
Yes, it works fine with that combination (arduino + platormio). I'm using many of these boards with nightdriverled.com Depending on the exact platformio/arduino/esp-idf you have installed, it can be a bit finicky. It's been a while since I've tried a stock build and I can't remember if I've modified mine, but your suspicion that they're basicaly a devkitc with a normal connector instead of a 2010 connector is correct. I just checked a project that builds and runs and it's stuper straight-forward.

```
➜  nightdriverstrip git:(s3-work) ✗ cat *yd.ini
[dev_yd]
extends         = dev_esp32-s3
board           = esp32-s3-devkitc-1
board_upload.flash_size = 8MB
board_build.partitions = default.csv
```

The boards are cheap. Just roll up your sleeves and tackle it.
